### DISCUSSION TITLE
Puppy needs some PR ie new releases

### DISCUSSION THREAD

**Discussion by mavrothal** [puppylinux-woof-CE/woof-CE#3376] [General]
The last “official” puppy was released to the public and distrowatch 2 years ago (Sep 2020).
3500 commits latter (!!!) is still the _latest official_ puppy.
The wider interest, as represented by distrowatch, is plummeting and the new forum has downgraded the all  “official” puppies to a practically invisible, single line adding to the problem.

So it appears that woof-CE/github has become a playground for developers and the forum a playground for “everything goes”. 
I think is a pity, as MX Linux that tops distrowatch, has many similarities to puppy. There is clearly an interest for this kind of distros and puppy can certainly fit in there, if was only released!

I know that you can not force anything in FOSS that the person does not have an itch for. However you can try to suggest…
So, please DO release one or more official puppies and announce updates as often as possible. Even if they are not perfect. For start upgrade the "unofficial" vanilla dpup to official, write a "catchy" announcement and call it "puppy 10" or 11 or...
Ask for a front page entry for each official puppy in the forum and if not given, move official forum elsewhere. Makes little sense for the "puppy" forum to focus on anything but puppy.

Bottom line: Do not sentence to invisibility (ie waste) all the work done here!

**Comment by dimkr** [comment]
> For start upgrade the "unofficial" vanilla dpup to official

The default dpup configuration in woof-CE (which is identical to Vanilla Dpup, except the "unbranded" name) produces something relatively minimal, and it will be a huge disappointment for people coming from @mrfricks' releases if it's advertised as "official".

I don't mind it if somebody else builds an "official" Puppy with dpup as the base, but I prefer to spend my (limited) time working the Debian 12 based dpup 10.0, which should be very similar to the Debian 11 based 9.x, but with a modernized Wayland/PipeWire/overlay stack with HiDPI and GTK+ 4 support. I don't have the time to implement all the things users expect from a worthy "official" Puppy, like piles of built-in applications, quickpet and so on.

Unless this "official" Puppy is done right, it can be dangerous and create even more fragmentation. Many Puppy users stick to an old Puppy release which works for them, so they don't receive any bug or security fixes. They learn how to live with issues or work around them, while developers (us) don't receive bug reports and feedback on new features. Some Puppy users even flee to "Puppy adjacent" distros, because they don't see an upgrade path, or need a distro that works on newer hardware. Unless that imaginary Puppy 10 is **so** good it becomes the new go-to Puppy to all the people who stick to Ubuntu 16.04, 18.04 and 20.04 based Puppy releases, it will be very hard to create a healthy feedback loop (10.0 -> bug reports and feedback -> fixes -> 10.1, ...) and create consensus around this young leaf of the Puppy distro tree.

**Comment by mavrothal** [comment]
> Unless this "official" Puppy is done right

I would disagree. 
I think that waiting for the "right" one will be left with no one to offer it to.
An Ubuntu 22.04- and/or Slackware 15 -based pup is way overdue. 
Wayland or not, PPM or not is better than the desert there is now.
Peebee tries to put out stuff but is like you V-pups, but they are both not "official" and get little attention outside the 50-100 persons in the forum.
Puppy needs a front page release.

PS: I can appreciate that you may have no time for the hassle of a release, thus a "discussion" and not an "issue"

**Comment by dimkr** [comment]
@01micko Time for Slacko 8.0?

**Comment by peabee** [comment]
see #3542

**Comment by 01micko** [comment]
Slacko isn't what the users want. They want debian/ubuntu based. Slack package management is not for the feint of heart!
